tests/unit/lms/services/mailchimp_test.py

```diff
@@ -14,16 +14,33 @@
 
 
 class TestSend:
+    @pytest.mark.parametrize(
+        "total_annotations,courses,expected_subject",
+        [
+            (2, [sentinel.course_1, sentinel.course_2], "Hypothesis: 2 annotations in 2 courses yesterday"),
+            (2, [sentinel.course_1], "Hypothesis: 2 annotations in 1 course yesterday"),
+            (1, [sentinel.course_1], "Hypothesis: 1 annotation yesterday"),
+        ],
+    )
```

Comment:
Now that `total_annotations` and `courses` are used in an `{% if %}` in the template body they have to be in the template variables or rendering the template crashes.

It's a bit odd that it ends up being in the tests of `MailchimpService` that the instructor email digest template gets tested, but it's `MailchimpService` that actually renders the template